static/app/utils/replays/replayReader.tsx

```diff
@@ -93,7 +93,7 @@ export default class ReplayReader {
    * @returns Duration of Replay (milliseonds)
    */
   getDurationMS = () => {
-    return this.event.endTimestamp - this.event.startTimestamp;
```

Comment:
not even that far away, line 78+79 we're converting from ms to seconds and storing those values

```
this.event = {
      ...event,
      startTimestamp: startTimestampMS / 1000,
      endTimestamp: endTimestampMS / 1000,
```
ugh, need to bring back the precision here.